src/email.rs

```diff
@@ -113,21 +108,20 @@ fn send_email(recipient: &str, subject: &str, body: &str) -> AppResult<()> {
 
     match mailgun_config {
         Some(mailgun_config) => {
-            let mut transport = SmtpClient::new_simple(&mailgun_config.smtp_server)?
+            let transport = SmtpTransport::builder(&mailgun_config.smtp_server)
```

Comment:
```suggestion
            let transport = SmtpTransport::relay(&mailgun_config.smtp_server)
```

I saw your issue again https://github.com/lettre/lettre/issues/431 and this needs to be a relay, else it won't be configured by default to speak TLS. Sorry for not catching this earlier. We'll update the lettre docs to better show this.